standalone-metastore/metastore-server/src/main/sql/postgres/upgrade-4.0.0-alpha-1-to-4.0.0-alpha-2.postgres.sql

```diff
@@ -0,0 +1,374 @@
+SELECT 'Upgrading MetaStore schema from 4.0.0-alpha-1 to 4.0.0-alpha-2';
```

Comment:
Shouldn't this Postgres upgrade script be mostly empty (except version update) at the moment?

Reply:
Thanks for catching this.
Fixed